drivers/windows/R/batch.R

```diff
@@ -60,12 +90,17 @@ template_data_common <- function(config, path_root) {
     "ECHO Removing mapping {{drive}}\nnet use {{drive}} /delete /y",
     network_shares_data)
 
+  r_version <- config$r_version
+  if (config$platform != "linux") {
+    r_version <- version_string(config$r_version)
+  }
```

Comment:
What is going on here? Why a conversion into string only for not linux?

Reply:
The version_string converts "4.4.2" into "4_4_2" which windows wants for its "call setr64_4_4_2.bat", but linux does not, for its "module load R/4.4.2"

I'll make this more clear.

Reply:
Commented